wrappers/models_configurations/config_types.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+class ConfigTypes(str, Enum):
+    BASE = 'BASE'
```

Comment:
is base configuration something we want to have? do we expect a saved file by this name? if it's just a placeholder for the base class, I think a better solution would be using an abstract method "get_file_name() / get_model_type()".

Reply:
Removed `BASE`, I don't see real usage for it, and added an abstract method of `_get_config_file_name`